hash_practice/exercises.py

```diff
@@ -22,8 +40,22 @@ def valid_sudoku(table):
         Each element can either be a ".", or a digit 1-9
         The same digit cannot appear twice or more in the same 
         row, column or 3x3 subgrid
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n2)

```

Comment:
This would be correct (even given your current buggy implementation) if we were taking in arbitrary nxn grids, but as we are only taking in 3x3 grids, the run time is constant and thus O(1).